### PR TITLE
Only `require` *.js libs from *.js files, not *.es6 files

### PR DESCRIPTION
**Pull request by jwietelmann**

### CODE REVIEW

File: app/assets/javascripts/revelry.js [hunk 1]
```diff
@@ -1,9 +1,16 @@
+//= require es6shim
```
Comment (lintomat) [inline app/assets/javascripts/revelry.js]: Error: secure random number generation not supported by this browser
use chrome, FireFox or Internet Explorer 11

File: lib/revelry_core/version.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 module Revelry
   module Core
-    VERSION = "0.1.13.0"
+    VERSION = "0.1.14.0"
```
Comment (lintomat) [inline lib/revelry_core/version.rb]: Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: app/assets/javascripts/revelry.js (hunk 1 above)
Comment (lintomat) [inline app/assets/javascripts/revelry.js]: Expected changes or additions to a test file called [revelry_spec.js](https://github.com/revelrylabs/revelry_core/blob/5faa27c835c31ea19311299e6f37b2843f86adf6/spec/javascripts/revelry_spec.js)

File: lib/revelry_core/version.rb (hunk 2 above)
Comment (lintomat) [inline lib/revelry_core/version.rb]: Expected changes or additions to a test file called [version_spec.rb](https://github.com/revelrylabs/revelry_core/blob/5faa27c835c31ea19311299e6f37b2843f86adf6/spec/revelry_core/version_spec.rb)

**Comment by lintomat** [comment]
To relint, [click here](http://lintron.herokuapp.com/relint/revelrylabs/revelry_core/93).

File: revelry_core.gemspec [hunk 3]
```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |s|
   s.add_dependency "therubyracer", '>= 0.12' # There are compatibility issues between older ruby racers and new coffee-rails versions
   s.add_dependency "react-source", ">= 0.13"
   s.add_dependency "jquery-rails"
-  s.add_dependency "react-rails", "~> 1.3.0" # For the react_ujs support
+  s.add_dependency "react-rails", "1.3.1" # For the react_ujs support.
```
Comment (jwietelmann) [inline revelry_core.gemspec]: The react-rails team screwed us in a revision release again. Just pinning this down forever until we can move off of the Sprockets ecosystem altogether.

**Pull request merged**